music21/metadata/__init__.py

```diff
@@ -1184,9 +1192,9 @@ def composer(self):
         ...     )
         >>> md.composer
         'Beach, Mrs. H.H.A.'
-        >>> md.add('composer', 'Beach, Amy Marcy Cheney')
+        >>> md.add('composer', 'Beach, Amy Marcy Cheney') # we need a better example here
```

Comment:
yeah, this is a better example of Alternative Contributor Name than another composer.  Leave this as the simple example, and add a collaborative work as another example.  In the world of popular music, they're common.  I think I gave some examples earlier.  Here's one I'd really like to see represented in music21:

```
        >>> md = metadata.Metadata(
        ...     title='...(Iphigenia)',
        ...     composer='Shorter, Wayne',
        ...     )
        >>> md.add('composer', 'Spalding, Esperanza')
```
(https://www.npr.org/2021/12/10/1062355320/wayne-shorter-and-esperanza-spalding-crack-open-the-old-world-in-new-opera)

Reply:
Done.  Used your better example.